doc/api/errors.md

```diff
@@ -644,6 +644,17 @@ according to the encoding provided.
 Used by the `util.TextDecoder()` API when the encoding provided is not one of
 the [WHATWG Supported Encodings][].
 
+<a id="ERR_EVENTS_MAX_LISTENERS_TYPE"></a>
+### ERR_EVENTS_MAX_LISTENERS_TYPE
+
+Used when an attempt is made to set `require('events').defaultMaxListeners` to
```

Comment:
Would be nice to link this to the documentation of `EventEmitter.defaultMaxListeners`